### ISSUE TITLE
怎么构建模型

### ISSUE THREAD

**Issue by Weia** [zhaoyuzhi/Deep-Convolutional-Network-Cascade-for-Facial-Point-Detection#2]
模型有三个阶段，感觉看你的代码，是一个一个阶段的训练的吗？

**Comment by zhaoyuzhi** [comment]
Yes, it is layer-by-layer trained. To be more precisely, it is one network by one network trained. Then you can use 'def' function to connect them. F1 is precise enough.

**Comment by zhaoyuzhi** [comment]
I just upload all the .py files and please see README. You will know my training method.
